harvester_e2e_tests/integration/test_vm_functions.py

```diff
@@ -99,6 +139,406 @@ def test_minimal_vm(api_client, image_id, unique_vm_name, wait_timeout):
         )
 
 
+@pytest.mark.p0
+@pytest.mark.virtualmachines
+@pytest.mark.parametrize("resource", [("cpu", 999999), ("mem", 999999), ("disk", 999999),
+                                      ("mem_and_cpu", 999999), ("mem_cpu_and_disk", 999999)],
+                         ids=['cpu', 'mem', 'disk', 'mem_and_cpu', 'mem_cpu_and_disk'])
+def test_create_vm_no_available_resources(resource, api_client, image_id, wait_timeout):
+    """Creates a VM with outlandish resources for varying elements (purposefully negative test)
```

Comment:
docstring should add refer to manual test case

Reply:
updated to include those elements in the docstring :smile: